phpcs.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<ruleset name="AMP Admanager PHPCS Ruleset">
+    <description>WordPress Coding Standards</description>
+
+    <rule ref="WordPress-Docs" />
+    <rule ref="WordPress-Core">
+        <exclude name="WordPress.Files.FileName.NotHyphenatedLowercase" />
+        <exclude name="WordPress.Files.FileName.InvalidClassFileName" />
+    </rule>
+    <rule ref="WordPress" />
+
+    <config name="testVersion" value="7.0-"/>
```

Comment:
Plugin readme mentions support for PHP `5.4+`, this should be similar to that.

Reply:
@thrijith Updated PHPCS test version as suggested